build/js/main.js

```diff
@@ -1034,6 +1047,11 @@ class Bluetooth extends Plugin {
         <div id="statusMessages" class="text grid__col grid__col--8-of-8"></div>
         `;
 
+        // Disable buttons
+        document.getElementById('BT_Pair').disabled = true;
+        document.getElementById('BT_Connect').disabled = true;
+        document.getElementById('BT_Disconnect').disabled = true;
+
```

Comment:
Same here

Better move those a little lower and bind them to a this.xyzButton value.